### ISSUE TITLE
WCAG 2.2 errata in text alternative 

### ISSUE THREAD

**Issue by rscano** [w3c/wcag#3486]
In text alternative definition there is a note: 

'<div class="note" role="note" id="issue-container-generatedID-149"><div role="heading" class="note-title marker" id="h-note-149" aria-level="3"><span>Note</span></div><p class="">Refer to <a href="https://www.w3.org/WAI/WCAG21/Understanding/conformance#text-alternatives">Understanding Text Alternatives</a> for more information.
</p></div>`

WCAG21 link should be WCAG22

**Comment by kfranqueiro** [comment]
Fixed by #4586.
